src/block/icon-label/style.js

```diff
@@ -26,6 +26,7 @@ const Styles = props => {
 
 	return (
 		<>
+			{ /* For the old icon gap */ }
```

Comment:
I believe you should be able to remove the `BlockCss` entry for the old `iconGap` here and the block should not error out.